code/src/components/Header.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const Header = () => {
+  const date = new Date();
+  const day = date.getDate();
+  const month = date.getMonth() + 1;
+  const year = date.getFullYear();
+  const currentDate = `${day}/${month}/${year}`;
```

Comment:
It's a little long code snippet that could probably be made shorter, but honestly I understand everything of what is happening here so I like it!